### PR TITLE
hooking everything together

### PR DESCRIPTION
**Pull request by kyleferguson**
brizo-working

### CODE REVIEW

**Comment by kyleferguson** [comment]
Basic design added, ready for demo :)
brizo-design

File: .env.example [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 BRIZO_MYSQL_USER=root
 BRIZO_MYSQL_PASSWORD=secret
 BRIZO_MYSQL_HOST=127.0.0.1
 BRIZO_MYSQL_PORT=33062
 BRIZO_MYSQL_NAME=brizo
 
-OAUTH_GITHUB_CLIENT_ID=exampleClientId
-OAUTH_GITHUB_CLIENT_SECRET=exampleClientSecret
+OAUTH_GITHUB_CLIENT_ID=3107e9dc652f2ffb95a7
+OAUTH_GITHUB_CLIENT_SECRET=cddd23dc61e6faa0b1d28233d9e9b958272757ad
```
Comment (jlaswell) [inline .env.example:10]: Committed the Client Id and Secret here.

File: ui/src/app/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
```
Comment (jlaswell) [inline ui/src/app/dashboard/dashboard.component.ts]: Unused import

**Review by jlaswell — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (kyleferguson) [reply .env.example:10]: Intentional. I setup a `brizo-dev` application for gentux for dev simplicity. Probably bad practice for a public repo but figured we could change it easily if it becomes an issue.

**Comment by jlaswell** [comment]
lgtm

**Review by jlaswell — APPROVED** [review]

**Pull request merged**